pybox/utils.py

```diff
@@ -0,0 +1,7 @@
+__all__ = (
+    'singleton',
+)
+
+
+def singleton(cls):
```

Comment:
let's do a common singleton which allows to save common syntax of class instantiating in other pull